README.md

```diff
@@ -508,11 +520,12 @@ Authentication tokens are passed in on the `context.auth` object. If no authenti
 
 ### Action Retries
 
-If you expect that your action might hit the 30 second time limit, consider adding retry support with `context.retry`. You **must** return this in order for your action to retry- it will not do so automatically if it hits the timeout. See a full action example using this [here](https://github.com/clay-run/action-package-apis/blob/master/src/aws_textract_pdf/aws_textract_pdf.js#L81).
+If you expect that your action might hit the 30 second time limit, consider adding retry support with `context.retry`. You **must** return this in order for your action to retry- it will not do so automatically if it hits the timeout. See a full action example using this [here](https://github.com/clay-run/action-package-apis/blob/master/src/aws_textract_pdf/aws_textract_pdf.js#L71-L74).
 
 A quick reference:
 ```js
-  context.retry({
-    message: 'message here'
+  return context.retry({
+    message: 'message here',
+    retryType: context.status.RETRY
```

Comment:
do we have other retry types? I'm not sure this is needed

Reply:
We can get rid of `retryType` but we would also need to remove that on `clay-action-client`

Reply:
https://github.com/clay-run/clay-action-client/pull/13

done

Reply:
approved and merged the removal of retry types from clay-action-client